lib/public/components/common/selection/infoLoggerButtonGroup/CopyToClipboardComponent.js

```diff
@@ -41,6 +41,56 @@ export class CopyToClipboardComponent extends StatefulComponent {
         this.notify();
     }
 
+    /**
+     * Checks if the clipboard is available and returns an object with availability status and a message.
+     *
+     * @returns {{ available: boolean, message: string }}
+     * - `available`: True if the clipboard is available; false otherwise.
+     * - `message`: A reason if the clipboard is not available, otherwise an empty string.
```

Comment:
If message do not apply (meaning: it makes no sense to have a message, in the case the clipboard is available) you should rather use null or undefined (basically return `{ available: true }`).

Another possibility is to have a function like `checkClipboardAvailability` that throws an error if one of the check fails, and you use the error message as message.